### ISSUE TITLE
API Doc Link not working

### ISSUE THREAD

**Issue by Yi-Jacob** [mempool/mempool#1703]
### Description

The "https://mempool.space/api/blocks-extras/736915" link does not work under the API/blocks/ get-blocks-extras

#### Version

v1?

### Steps to reproduce

Just need to add the version number to the api get request ie. "https://mempool.space/api/v1/blocks-extras/736915"

### Expected behaviour

The old linkshould have given a 200 status code and shown the extra data.

### Actual behaviour

The old link gave a 404 message but not when you add the version number

### Screenshots

https://user-images.githubusercontent.com/96788421/169660078-ec0bc6d7-19d3-496f-8535-5f9a8a1a16c1.mov

#### Device or machine

macbook/macOs

**Comment by ayanamidev** [comment]
@Yi-Jacob Yes, this was already recognized by https://github.com/mempool/mempool/pull/1691#discussion_r878651634 , seems like mempool.space is misconfigured.

**Comment by wiz** [comment]
@hunicus looks like we're missing /v1/ here
